base/setup/lib/utils/filesup.c

```diff
@@ -27,35 +27,32 @@
 static
 NTSTATUS
 SetupCreateSingleDirectory(
-    IN PCWSTR DirectoryName)
+    _In_ PCWSTR DirectoryName)
 {
+    NTSTATUS Status;
+    UNICODE_STRING PathName;
     OBJECT_ATTRIBUTES ObjectAttributes;
     IO_STATUS_BLOCK IoStatusBlock;
-    UNICODE_STRING PathName;
     HANDLE DirectoryHandle;
-    NTSTATUS Status;
 
-    if (!RtlCreateUnicodeString(&PathName, DirectoryName))
-        return STATUS_NO_MEMORY;
+    RtlInitUnicodeString(&PathName, DirectoryName);
 
     if (PathName.Length > sizeof(WCHAR) &&
```

Comment:
nit: This `PathName.Length > sizeof(WCHAR)` is not related to `sizeof(WCHAR)` but to `2` characters that can be removed, so the expression should probably be `PathName.Length > 2`

Reply:
`2 * sizeof(WCHAR)`, but yes, but in that case I _could_ use a `>=` instead?